docs/channels.md

```diff
@@ -0,0 +1,103 @@
+
+* `#announcements`: New Minecraft releases and important Fabric news is announced in this channel.
+
+* `#live-update-updates`: Information on the Fabric update process and how it adjusts to Minecraft updates is posted here, along with information we dig up during that process.
+
+* `#moderator-messages`: Important communications from the moderation team, directed at the community as a whole. We suggest you avoid muting this channel, for the sake of staying up to date on community policies and not being caught out by changes.
+
+## Showcase Category
+
+This category is where you go to show off your Fabric-related projects. Please note that this category has only one discussion channel — the others should not be used for conversation.
+
+All projects posted in these channels need to be related to Fabric somehow — for example: mods and libraries, or services for Fabric users.
+
+* `#new-releases`: You can post about the initial public release of your new projects here. We'll also allow posts for the first public release of a completely reworked project.
+
+* `#update-releases`: You can post when you release an update to a project that's already been released. Do not post newly-released projects here.
+
+* `#showcase`: This channel can be used to showcase the things you're working on - for example, a new feature you've added to a mod. As many members of our community are not programmers, we ask that you post in-game screenshots and videos where possible instead of code — although we do recognise that some projects can only be showcased through code.
+
+* `#showcase-discussion`: Use this channel to discuss anything that's been posted in any of the other three channels. Please note that **this is not an off-topic channel** — we have more than one of those in the community category if you wish to talk about something else.
+
+## Community Category
+
+* `#player-general`: This channel is a space to talk about Fabric as a player, as well as somewhere you can talk about Fabric mods and modpacks. It's also suitable for discussions based around server administration, hosting and working with the JVM, as well as looking-for-group style posts by people that are looking for somewhere to play. This is not a space for developers — see the mod development category for that.
+
+* `#player-support`: This is where you go if your game is crashing with Fabric installed — you can ask about help with specific mods, with Fabric itself, or just to try and figure out why your game isn't working.
+
+* `#general-discussion` & `#general-discussion-2`: Our off-topic community hangs out in these channels. These are **discussion channels**, which means that **they are not meant for dumping images, videos, memes, shitposts or other media**. Please read the channel before you post, and try not to disrupt an ongoing conversation — if one of the channels is busy, you can always try the other one.
```

Comment:
Maybe add that the media is OK as long as it is at least somewhat relevant to the discussion, or is done with the purpose of discussing it? Of course just dumping things would still be disallowed.

Reply:
I think I'd be happy to add this after the bold text in that quote:

`(although you can post media if it actually contributes to the current discussion, or is intended to start a new one)`

Reply:
Yep, OK.